pages/common/uudecode.md

```diff
@@ -0,0 +1,8 @@
+# uudecode
+
+> Decode files encoded by `uuencode`.
+> More information: <https://manned.org/uudecode>.
+
+- Decode a file that was create with `uuencode` into the current directory:
+
+`uudecode {{path/to/file}}`
```

Comment:
```suggestion
- Decode a file that was encoded with `uuencode` and print the result to stdout:

`uudecode {{path/to/encoded_file}}`

- Decode a file that was encoded with `uuencode` and write the result to a file:

`uudecode -o {{path/to/decoded_file}} {{path/to/encoded_file}}`
```
Maybe we should keep it consistent with the `uuencode` page.

Reply:
Also, the file name of the encoded file has to be the one, used when encoding the file. How should we mention that?

Reply:
Why are there duplicate commands